bin/compile-requirements.sh

```diff
@@ -4,7 +4,7 @@ set -exo pipefail
 
 export CUSTOM_COMPILE_COMMAND="make compile-requirements"
 if [[ "$1" == "--upgrade" ]]; then
-    pip-compile --generate-hashes --reuse-hashes --upgrade --upgrade-package 'django<2.3' requirements.in
+    pip-compile-multi --generate-hashes prod --generate-hashes dev --upgrade
 else
-    pip-compile --generate-hashes --reuse-hashes requirements.in
+    pip-compile-multi --generate-hashes prod --generate-hashes dev
```

Comment:
If my host Python version is different that Docker's, will this still produce the correct results? I know some sub-dependencies get installed depending on Python version.

Reply:
Yeah, this is the thing between running this all in the container or living with a mix of local and containerised... I didn't like the idea of having pip-tools itself in the dependency manifests, in part because a stale pip-tools in hte project created a circular dependency problem where I couldn't update pip tools to update other deps. So, I moved it out, but can move it back in to run in the container, sure

In theory, if you get an inviable dep mix because you updated them on, say, 3.9 on your host, the 3.7 container should fail to install. However, this isn't ideal, I know, so let's stabilise it

- [ ] @stevejalim move to run in the container